vitrina/orgs/forms.py

```diff
@@ -191,15 +192,19 @@ class Meta:
         )
 
     def __init__(self, *args, **kwargs):
+        user = kwargs.pop("user")
         super().__init__(*args, **kwargs)
-        org_instance = self.instance if self.instance and self.instance.pk else None
-        button = _("Redaguoti") if org_instance else _("Sukurti")
-        parent = self.instance.get_parent()
+
+        parent = getattr(self.instance, "get_parent", lambda: None)()
```

Comment:
Nitpick: I think this is more readable
```suggestion
        parent = self.instance.get_parent() if hasattr(self.instance, "get_parent") else None
```